test/fast/Running "nvm use" should strip windows line-endings from nvmrc.

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
+die () { echo $@ ; cleanup ; exit 1; }
+
+cleanup() {
+  rm -rf .nvmrc
+}
+
+. ../../nvm.sh
+
+# Use a known different version, first.
+nvm install v5.6.0 >/dev/null 2>/dev/null
```

Comment:
"fast" tests definitely can't do a full install, that would be too slow - I think it's enough to add a nonexistent version to `.nvmrc` like "0.20.0" and just take this as a given.